.github/workflows/tests.yml

```diff
@@ -364,16 +364,16 @@ jobs:
 
   api_ee_mysql:
     runs-on: ubuntu-latest
-    needs: [changes, lint, typescript, unit_back, unit_front]
+    needs: [changes, build, typescript, unit_back, unit_front]
     name: '[EE] API Integration (mysql:latest, client: ${{ matrix.db_client }}, node: ${{ matrix.node }}, shard: ${{ matrix.shard }})'
     if: needs.changes.outputs.backend == 'true' && github.event.pull_request.head.repo.full_name == github.repository && !(github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]')
     env:
       STRAPI_LICENSE: ${{ secrets.strapiLicense }}
     strategy:
       matrix:
-        node: [16, 18, 20]
+        node: [18, 20]
```

Comment:
why removing node16 from the matrix from the mysql action?

Reply:
Yep, should be merged in just a few minutes and then this change will disappear 😆